docs/openapi/components/schemas/common/MinesActPermit.yml

```
@@ -0,0 +1,177 @@
+    title: Mine Operation Status
+    description: >-
+      Layer one in the hierarchy of identifying mine status. Mines status can be (1) Abandoned (2) Closed (3) Not Started (4) Operating.
+    type: string
+    $linkedData:
+      term: mine_operation_status
+      '@id': https://w3id.org/traceability#mine_operation_status
+  mine_operation_status_reason:
+    title: Mine Operation Status Reason
+    description: >-
+      Layer two in the hierarchy for identifying mine status. Reasons for Closed Mines (1) Closed for Care and Maintenance, (2) Closed for Reclamation and (3) Closed for Unknown. Reasons for Operating (1) Operating Seasonal, or (2) Operating Year Round
+    type: string
+    $linkedData:
+      term: mine_operation_status_reason
+      '@id': https://w3id.org/traceability#mine_operation_status_reason
+  mine_operation_status_sub_reason:
+    title: Mine Operation Status Sub Reason
+    description: >-
+      Layer three in the hierarchy for identifying mine status. Mine Operation Reasons for "Closed for Reclamation" (1) Long-Term Maintenance (2) Long-Term Maintenance and Water Treatment or (3) Permit Release Pending.
+    type: string
+    $linkedData:
+      term: mine_operation_status_sub_reason
+      '@id': https://w3id.org/traceability#mine_operation_status_sub_reason
+  mine_commodity:
+    title: Mine Commodity
+    description: >-
+      Lists the name(s) of the commodities being mined.
+    type: string
+    $linkedData:
+      term: mine_commodity
+      '@id': https://w3id.org/traceability#mine_commodity
+  mine_disturbance:
+    title: Mine Disturbance
+    description: >-
+      Specifies if mine is above ground (surface) or below ground (sub-surface).
+    type: string
+    $linkedData:
+      term: mine_disturbance
+      '@id': https://w3id.org/traceability#mine_disturbance
+  mine_no:
+    title: Mine Number
+    description: >-
+      (7-digit) Unique identifier assigned to a specific mine or mining operation to differentiate and track individual mines within a jurisdiction.
+    type: string
+    $linkedData:
+      term: mine_no
+      '@id': https://w3id.org/traceability#mine_no
+  issue_date:
+    title: Permit Start Date
+    description: >-
+      (YYYY-MM-DD) The original issue date of the permit.
+    type: string
+    $linkedData:
+      term: issue_date
+      '@id': https://w3id.org/traceability#issue_date
+  latitude:
+    title: Latitude
+    description: >-
+      (00.0000000) The geographic coordinate representing the north-south position of a specific location related to the administrative building at the mining operation or mine site.
+    type: string
+    $linkedData:
+      term: latitude
+      '@id': https://w3id.org/traceability#latitude
+  longitude:
+    title: Longitude
+    description: >-
+      (-000.0000000) The geographic coordinate that represents the east-west position of a specific location related to the administrative building at the mining operation or mine site.
```

Either this change, or `representing` should change to `that represents`
```suggestion
      (00.0000000) The geographic coordinate representing the north-south position of a specific location related to the administrative building at the mining operation or mine site.
    type: string
    $linkedData:
      term: latitude
      '@id': https://w3id.org/traceability#latitude
  longitude:
    title: Longitude
    description: >-
      (-000.0000000) The geographic coordinate representing the east-west position of a specific location related to the administrative building at the mining operation or mine site.
```